Formula/m/mpd.rb

```diff
@@ -4,7 +4,7 @@ class Mpd < Formula
   url "https://github.com/MusicPlayerDaemon/MPD/archive/refs/tags/v0.23.13.tar.gz"
   sha256 "c002fd15033d791c8ac3dcc009b728b0e8440ed483ba56e3ff8964587fe9f97d"
   license "GPL-2.0-or-later"
-  revision 1
+  revision 2
```

Comment:
I'm just a newbie here but the change to be effective needs a rebuild of MPD and since the updated formulea will provide now updated MPD binary shouldn't it have a different revision or any other mark of difference vs the previous formulea?

Reply:
A revision bump is not necessary for a rebuild. We only bump the revision number when it's necessary, because that would force every `mpd` user to download and install the new update, for which we have received some complaints in the past.

Without a revision bump, it's still possible to install the rebuilt formula with `brew reinstall mpd` after this gets merged.